### PR TITLE
fix(menu): add RTL chevron menu icon

### PR DESCRIPTION
**Pull request by V-Ahmetvaliev**
also change ion to nb icon

### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [x] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:

### CODE REVIEW

**Comment by nnixaa** [comment]
@V-Ahmetvaliev @yggg shouldn't we subscribe to changes and change the template based on it?

File: src/framework/theme/components/menu/menu-item.component.html [hunk 1]
```diff
@@ -42,8 +42,10 @@
    href="#">
   <i class="menu-icon {{ menuItem.icon }}" *ngIf="menuItem.icon"></i>
   <span class="menu-title">{{ menuItem.title }}</span>
-  <i class="ion chevron" [class.ion-chevron-left]="!menuItem.expanded"
-     [class.ion-chevron-down]="menuItem.expanded"></i>
+  <i class="ion chevron" [ngClass]="{ 'nb-chevron-right': isLtr && !menuItem.expanded,
```
Comment (yggg) [inline src/framework/theme/components/menu/menu-item.component.html]: Please, don't forget to change `nb-chevron-left` to match ltr direction  🙂

File: src/framework/theme/components/menu/menu.component.ts [hunk 2]
```diff
@@ -50,8 +51,20 @@ export class NbMenuItemComponent implements DoCheck, AfterViewInit, OnDestroy {
 
   private alive = true;
   toggleState: NbToggleStates;
+  isLtr$: Observable<boolean>;
+  isRtl$: Observable<boolean>;
 
-  constructor(private menuService: NbMenuService) {}
+  constructor(private menuService: NbMenuService,
+              private directionService: NbLayoutDirectionService) {
+    this.isLtr$ = this.directionService.onDirectionChange()
```
Comment (yggg) [inline src/framework/theme/components/menu/menu.component.ts]: It's better to move initialization code into lifecycle method, such as `ngOnInit`.

Comment (yggg) [inline src/framework/theme/components/menu/menu.component.ts]: It's better to move initialization code into lifecycle method, such as `ngOnInit`.

File: src/framework/theme/components/menu/menu-item.component.html [hunk 3]
```diff
@@ -43,8 +43,11 @@
    href="#">
   <i class="menu-icon {{ menuItem.icon }}" *ngIf="menuItem.icon"></i>
   <span class="menu-title">{{ menuItem.title }}</span>
-  <i class="ion chevron" [class.ion-chevron-left]="!menuItem.expanded"
-     [class.ion-chevron-down]="menuItem.expanded"></i>
+  <i class="chevron"
+     [class.nb-arrow-right]="(isRtl$ | async) && !menuItem.expanded"
```
Comment (yggg) [inline src/framework/theme/components/menu/menu-item.component.html]: Add [`BREAKING CHANGE:`](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md#footer) commit for this change

**Pull request merged**